src/utils/AstUtils.ts

```diff
@@ -16,11 +17,13 @@ export module AstUtils {
 
     export function getFunctionName(node: ts.CallExpression | ts.NewExpression): string {
         const expression: ts.Expression = node.expression;
-        let functionName: string = (<any>expression).text;
-        if (functionName === undefined && (<any>expression).name) {
-            functionName = (<any>expression).name.text;
+        if ('text' in expression) {
```

Comment:
This was an interesting one. Just using `expression.name.getText()` caused a whole bunch of tests to fail, so this does need to look for a `text` property on the expression, but it can't look for a `getText()` method, because that caused a bunch of different tests to fail.